rln/src/public.rs

```diff
@@ -118,8 +118,7 @@ impl RLN<'_> {
         Ok(())
     }
 
-    //TODO: change to set_leaves_from(index, input_data)
-    pub fn set_leaves<R: Read>(&mut self, mut input_data: R) -> io::Result<()> {
+    pub fn set_leaves_from<R: Read>(&mut self, index: usize, mut input_data: R) -> io::Result<()> {
```

Comment:
Any considerations w.r.t order of arguments? index or input_data first e.g.

Reply:
I believe having the index first in the set of arguments is idiomatic rust - see [Vec insert](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.insert)